From 0efce6c76b687f646979cf1efb5bd97f1d4536fe Mon Sep 17 00:00:00 2001
From: ranlily0830 <notkeyno6@gmail.com>
Date: Mon, 27 Apr 2026 21:21:01 +0800
Subject: [PATCH] Replace password patch's censort to '*'.

---
 dmenu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dmenu.c b/dmenu.c
index 77db1fd..2fb8048 100644
--- a/dmenu.c
+++ b/dmenu.c
@@ -167,7 +167,7 @@ drawmenu(void)
 	drw_setscheme(drw, scheme[SchemeNorm]);
 	if (passwd) {
 	        censort = ecalloc(1, sizeof(text));
-		memset(censort, '.', strlen(text));
+		memset(censort, '*', strlen(text));
 		drw_text(drw, x, 0, w, bh, lrpad / 2, censort, 0);
 		free(censort);
 	} else drw_text(drw, x, 0, w, bh, lrpad / 2, text, 0);
-- 
2.54.0

